test/lazy/test_extract_compiled_graph.py

```diff
@@ -80,19 +82,20 @@ def forward(self, a, b):
         c = a + b
         return a - b, c, a + 1, c
 
-class ModuleWithLTCFallback(nn.Module):
-    """
-    Cover the case that the Fx graph contains LTC fallbacks. In that case,
-    an exception should be thrown.
-    """
-    def __init__(self):
-        super(ModuleWithLTCFallback, self).__init__()
+@contextmanager
+def force_fallback_ctx_mgr(fallback_op):
+    config.set_force_fallback(fallback_op)
+    try:
+        yield None
+    finally:
+        config.set_force_fallback("")
```

Comment:
maybe better to first do
_orig_force_fallback = config.get_force_fallback()
finally
config.set_force_fallback(_orig_force_fallback)
?

probably unlikely to use multiple values though..

Reply:
yep, that's better.